annotations/fenix/metrics/tabs_tray.inactive_tabs_collapsed/README.md

```diff
@@ -0,0 +1,6 @@
+---
+tags:
+  - Tabs
+---
+
+A user tapped the "Inactive tabs" header to collapse the group of inactive tabs.
```

Comment:
This is just a duplication of the metric description, better to just leave it blank if you don't have anything to add here. 

Just to clarify, here's how the Glean Dictionary currently renders things (the description on top comes directly from the metrics.yaml files defined in Fenix):

https://dictionary.telemetry.mozilla.org/apps/fenix/metrics/tabs_tray_closed?channel=org.mozilla.firefox&ping=events

```suggestion
```

Reply:
Thank you! Will keep this in mind for future work.